src/api/tokenList/TokenListApiMock.ts

```diff
@@ -34,6 +36,10 @@ export class TokenListApiMock extends GenericSubscriptions<TokenDetails[]> imple
     this._tokenList = tokenList
     this.triggerSubscriptions(tokenList)
   }
+
+  public setListReady(state: boolean): void {
+    this.isListReady = state
+  }
```

Comment:
Should this not also `triggerSubscriptions`? I'm not sure `isListReady` will be picked up by `useOrders` hook immediately without.

Reply:
It is picked up in the beginning but maybe your right